packages/editor-ui/src/views/NodeView.v2.vue

```diff
@@ -1502,6 +1505,31 @@ function unregisterCustomActions() {
 	unregisterCustomAction('showNodeCreator');
 }
 
+async function createSampleSubworkflow() {
+	resetWorkspace();
+
+	canvasStore.startLoading();
+
+	canvasStore.setLoadingText(i18n.baseText('nodeView.loadingTemplate'));
+
+	workflowsStore.currentWorkflowExecutions = [];
+	executionsStore.activeExecution = null;
+
+	workflowsStore.setConnections(SAMPLE_SUBWORKFLOW_WORKFLOW.connections);
+
+	await addNodes(SAMPLE_SUBWORKFLOW_WORKFLOW.nodes, { trackHistory: true });
+
+	await workflowsStore.getNewWorkflowData('My Sub-Workflow', projectsStore.currentProjectId);
+
+	await workflowHelpers.saveAsNewWorkflow();
+
+	uiStore.stateIsDirty = true;
+
+	canvasStore.stopLoading();
+
+	fitView();
+}
```

Comment:
Can we also reset the URL parameters once we land here and create the sub-workflow? Same for the old canvas

Reply:
Done!